README.adoc

```diff
@@ -35,6 +36,33 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+Using npm:
+
+ $ npm i -g npm
+ $ npm i --save asciidoctor.js
+ $ npm i --save asciidoctor-reveal.js
+
+In Node.js:
```

Comment:
You provide the filename and the content, which is good but then how do you invoke the actual rendering? It is unclear to me.